dotcom-rendering/src/lib/dailyArticleCount.ts

```diff
@@ -14,10 +16,9 @@ export const DailyArticleCountKey = 'gu.history.dailyArticleCount';
 
 // Returns undefined if no daily article count in local storage
 export const getDailyArticleCount = (): DailyArticleHistory | undefined => {
-	// eslint-disable-next-line no-restricted-syntax -- FIXME-libs-storage
-	const dailyCount = localStorage.getItem(DailyArticleCountKey);
+	const dailyCount = storage.local.get(DailyArticleCountKey);
 
-	if (!dailyCount) {
+	if (!dailyCount || typeof dailyCount !== 'string') {
```

Comment:
If we’re using `@guardian/libs`… 

```suggestion
	if (!isString(dailyCount)) {
```

Reply:
oh good point! it looks like `@guardian/libs` parses the value as well, so `isString` is always false (as the value is probably an array). I've refactored to just assume the value we get back from `local.get` is an array. Interested to hear what you think!